.github/scripts/prepare-test-monorepo.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -e
+
+PREFIX="$1"
+REPOS="${PREFIX}/repos"
+WORKD="${PREFIX}/top"
+
+mkdir -p "${REPOS}/top.git"
+mkdir -p "${REPOS}/foo.git"
+mkdir -p "${REPOS}/bar.git"
+mkdir -p "${WORKD}"
+
+(cd $REPOS/top.git && git init --bare)
+(cd $REPOS/foo.git && git init --bare)
+(cd $REPOS/bar.git && git init --bare)
+
+pushd "$WORKD"
+
+git init;
+echo "TOP" > README.txt;
+cat > monorepo-builder.yml <<!
+parameters:
+  package_directories:
+    - 'packages'
+  directories_to_repositories:
+    packages/foo: "file://$REPOS/foo.git"
+    packages/bar: "file://$REPOS/bar.git"
```

Comment:
I sent it yestereday, and haven't noticed on mobile phone that Github didn't sent it beucase of one more confirmation button.
I've noticed that just now. I'm sorry I haven't seen it earlier

Reply:
This is still work in progress. GitHub repositories will need additional setup, such as some kind of authorization (push access!) for the test, so for now I'm using file repositories. I think it will be better to have example GitHub repositories owned by the symplify org.

Reply:
Also, how can we provide always empty (fresh) GitHub repository? I mean some kind of consistent state when test starts? What about multiple consurent PRs pushing to single GitHub repository (after split and push we should ideally verify the content of the target repo, which can change in the meantime due to other PR being tested).